process/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Flask==1.0.2
+google-cloud-pubsub
+google-cloud-bigquery
+requests
```

Comment:
likewise not needed if `requests` is not used in `main.py`

Reply:
Removed along with unused imports